Octokit/Models/Response/User.cs

```diff
@@ -23,6 +27,13 @@ public User(string avatarUrl, string bio, string blog, int collaborators, string
         /// </summary>
         public bool SiteAdmin { get; protected set; }
 
+        /// <summary>
+        /// LDAP Binding (GitHub Enterprise only)
+        /// </summary>
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "Dn")]
+        [SuppressMessage("Microsoft.Naming", "CA1709:IdentifiersShouldBeCasedCorrectly", MessageId = "Dn")]
+        public string LdapDn { get; protected set; }
```

Comment:
> I didnt realise we could "control" what words are allowed.

That's done via the `CustomDictionary.xml` file (it's in the "Build" solution folder). I'd rather not add `DN` as a custom word though and use the full name for the C# API and use the `ParameterAttribute` to control the JSON name.